apps/src/StudioApp.js

```diff
@@ -1055,7 +1055,7 @@ StudioApp.prototype.showInstructionsDialog_ = function (level, autoClose) {
 
   var hideFn = _.bind(function () {
     // Set focus to ace editor when instructions close:
-    if (this.editCode && this.editor && !this.editor.currentlyUsingBlocks) {
+    if (this.editCode && this.editor && !this.editor.session.currentlyUsingBlocks) {
```

Comment:
Do we need to check for `editor.session` first here? (like you do in `resizeToolbarHeader()`?)